### ISSUE TITLE
Debug doesn't work

### ISSUE THREAD

**Issue by Ygg01** [intellij-rust/intellij-rust#10006]
## Environment

* **IntelliJ Rust plugin version:** 0.4.186.5143-223
* **Rust toolchain version:** 1.66.1 (90743e729 2023-01-10) x86_64-pc-windows-msvc
* **IDE name and version:** CLion 2023.1 EAP (CL-231.4840.362)
* **Operating system:** Windows 10 10.0
* **Macro expansion:** enabled

## Problem description
Debugging doesn't work. There are some errors written and the IDE stops

## Steps to reproduce
1. Put a breakpoint
2. Start debugging
3. Expect  data get exception and following screen

![clion64_bhYhBlomvE](https://user-images.githubusercontent.com/1146204/213432569-6e6c5835-132a-4f96-a8c8-04222210b972.png)

```
java.lang.NoSuchMethodError
	at java.base/jdk.internal.reflect.GeneratedConstructorAccessor94.newInstance(Unknown Source)
	at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.base/java.lang.reflect.Constructor.newInstanceWithCaller(Constructor.java:499)
	at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:480)
	at java.base/java.util.concurrent.ForkJoinTask.getThrowableException(ForkJoinTask.java:564)
	at java.base/java.util.concurrent.ForkJoinTask.reportExecutionException(ForkJoinTask.java:604)
	at java.base/java.util.concurrent.ForkJoinTask.get(ForkJoinTask.java:981)
	at com.intellij.util.ConcurrencyUtil.manifestExceptionsIn(ConcurrencyUtil.java:256)
	at com.intellij.codeInsight.daemon.impl.DaemonCodeAnalyzerImpl.lambda$queuePassesCreation$19(DaemonCodeAnalyzerImpl.java:1040)
	at com.intellij.openapi.application.TransactionGuardImpl.runWithWritingAllowed(TransactionGuardImpl.java:209)
	at com.intellij.openapi.application.TransactionGuardImpl.access$100(TransactionGuardImpl.java:21)
	at com.intellij.openapi.application.TransactionGuardImpl$1.run(TransactionGuardImpl.java:191)
	at com.intellij.openapi.application.impl.ApplicationImpl.runIntendedWriteActionOnCurrentThread(ApplicationImpl.java:831)
	at com.intellij.openapi.application.impl.ApplicationImpl$3.run(ApplicationImpl.java:447)
	at com.intellij.openapi.application.impl.FlushQueue.doRun(FlushQueue.java:79)
	at com.intellij.openapi.application.impl.FlushQueue.runNextEvent(FlushQueue.java:122)
	at com.intellij.openapi.application.impl.FlushQueue.flushNow(FlushQueue.java:41)
	at java.desktop/java.awt.event.InvocationEvent.dispatch(InvocationEvent.java:318)
	at java.desktop/java.awt.EventQueue.dispatchEventImpl(EventQueue.java:786)
	at java.desktop/java.awt.EventQueue$3.run(EventQueue.java:737)
	at java.desktop/java.awt.EventQueue$3.run(EventQueue.java:731)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:399)
	at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:86)
	at java.desktop/java.awt.EventQueue.dispatchEvent(EventQueue.java:756)
	at com.intellij.ide.IdeEventQueue.defaultDispatchEvent(IdeEventQueue.java:934)
	at com.intellij.ide.IdeEventQueue.lambda$_dispatchEvent$9(IdeEventQueue.java:769)
	at com.intellij.openapi.application.impl.ApplicationImpl.withoutImplicitRead(ApplicationImpl.java:1433)
	at com.intellij.ide.IdeEventQueue._dispatchEvent(IdeEventQueue.java:769)
	at com.intellij.ide.IdeEventQueue.lambda$dispatchEvent$5(IdeEventQueue.java:449)
	at com.intellij.openapi.progress.impl.CoreProgressManager.computePrioritized(CoreProgressManager.java:787)
	at com.intellij.ide.IdeEventQueue.lambda$dispatchEvent$6(IdeEventQueue.java:448)
	at com.intellij.openapi.application.TransactionGuardImpl.performActivity(TransactionGuardImpl.java:105)
	at com.intellij.ide.IdeEventQueue.performActivity(IdeEventQueue.java:628)
	at com.intellij.ide.IdeEventQueue.lambda$dispatchEvent$7(IdeEventQueue.java:446)
	at com.intellij.openapi.application.impl.ApplicationImpl.runIntendedWriteActionOnCurrentThread(ApplicationImpl.java:831)
	at com.intellij.ide.IdeEventQueue.dispatchEvent(IdeEventQueue.java:492)
	at java.desktop/java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:207)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:128)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:117)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:113)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:105)
	at java.desktop/java.awt.EventDispatchThread.run(EventDispatchThread.java:92)
Caused by: java.lang.NoSuchMethodError: 'void org.intellij.markdown.parser.MarkdownParser.<init>(org.intellij.markdown.flavours.MarkdownFlavourDescriptor)'
	at org.rust.lang.doc.psi.RsDocCommentElementType.doParseContents(RsDocCommentElementType.kt:35)
	at com.intellij.psi.tree.ILazyParseableElementType.parseContents(ILazyParseableElementType.java:50)
	at com.intellij.psi.impl.source.tree.LazyParseableElement.lambda$ensureParsed$2(LazyParseableElement.java:175)
	at com.intellij.psi.impl.DebugUtil.performPsiModification(DebugUtil.java:481)
	at com.intellij.psi.impl.source.tree.LazyParseableElement.ensureParsed(LazyParseableElement.java:174)
	at com.intellij.psi.impl.source.tree.LazyParseableElement.getFirstChildNode(LazyParseableElement.java:226)
	at com.intellij.psi.impl.source.tree.LazyParseableElement.getFirstChildNode(LazyParseableElement.java:24)
	at com.intellij.psi.impl.source.tree.RecursiveTreeElementWalkingVisitor$ASTTreeGuide.getFirstChild(RecursiveTreeElementWalkingVisitor.java:48)
	at com.intellij.psi.impl.source.tree.RecursiveTreeElementWalkingVisitor$ASTTreeGuide.getFirstChild(RecursiveTreeElementWalkingVisitor.java:35)
	at com.intellij.util.WalkingState.next(WalkingState.java:70)
	at com.intellij.util.WalkingState.walkChildren(WalkingState.java:58)
	at com.intellij.util.WalkingState.elementStarted(WalkingState.java:49)
	at com.intellij.psi.impl.source.tree.RecursiveTreeElementWalkingVisitor.visitNode(RecursiveTreeElementWalkingVisitor.java:86)
	at org.rust.lang.core.stubs.ItemSeekingVisitor.visitNode(StubImplementations.kt:1931)
	at com.intellij.psi.impl.source.tree.RecursiveTreeElementWalkingVisitor.visitComposite(RecursiveTreeElementWalkingVisitor.java:81)
	at com.intellij.psi.impl.source.tree.CompositeElement.acceptTree(CompositeElement.java:117)
	at org.rust.lang.core.stubs.ItemSeekingVisitor$Companion.containsItems(StubImplementations.kt:1938)
	at org.rust.lang.core.stubs.RsBlockStubType.shouldCreateStub(StubImplementations.kt:1872)
	at com.intellij.psi.stubs.DefaultStubBuilder$StubBuildingWalkingVisitor.createStub(DefaultStubBuilder.java:80)
	at com.intellij.psi.stubs.DefaultStubBuilder$StubBuildingWalkingVisitor.visitNode(DefaultStubBuilder.java:60)
	at com.intellij.psi.stubs.DefaultStubBuilder$StubBuildingWalkingVisitor.buildStubTree(DefaultStubBuilder.java:55)
	at com.intellij.psi.stubs.DefaultStubBuilder.buildStubTreeFor(DefaultStubBuilder.java:33)
	at com.intellij.psi.stubs.DefaultStubBuilder.buildStubTree(DefaultStubBuilder.java:22)
	at com.intellij.psi.stubs.StubTreeBuilder.lambda$buildStubTree$1(StubTreeBuilder.java:144)
	at com.intellij.psi.stubs.StubTreeBuilder.handleStubBuilderException(StubTreeBuilder.java:99)
	at com.intellij.psi.stubs.StubTreeBuilder.lambda$buildStubTree$3(StubTreeBuilder.java:144)
	at com.intellij.psi.impl.PsiManagerImpl.runInBatchFilesMode(PsiManagerImpl.java:457)
	at com.intellij.psi.stubs.StubTreeBuilder.buildStubTree(StubTreeBuilder.java:134)
	at com.intellij.psi.stubs.StubTreeBuilder.buildStubTree(StubTreeBuilder.java:92)
	at com.intellij.psi.stubs.StubTreeLoaderImpl.lambda$build$0(StubTreeLoaderImpl.java:74)
	at com.intellij.openapi.util.RecursionManager$1.computePreventingRecursion(RecursionManager.java:112)
	at com.intellij.openapi.util.RecursionGuard.doPreventingRecursion(RecursionGuard.java:42)
	at com.intellij.openapi.util.RecursionManager.doPreventingRecursion(RecursionManager.java:66)
	at com.intellij.psi.stubs.StubTreeLoaderImpl.lambda$build$1(StubTreeLoaderImpl.java:74)
	at com.intellij.openapi.vfs.newvfs.impl.VirtualFileImpl.computeWithPreloadedContentHint(VirtualFileImpl.java:96)
	at com.intellij.psi.stubs.StubTreeLoaderImpl.build(StubTreeLoaderImpl.java:63)
	at com.intellij.psi.stubs.StubTreeLoaderImpl.readOrBuild(StubTreeLoaderImpl.java:54)
	at com.intellij.psi.impl.source.PsiFileImpl.getStubTree(PsiFileImpl.java:634)
	at com.intellij.psi.impl.source.PsiFileImpl.getGreenStubTree(PsiFileImpl.java:956)
	at com.intellij.psi.impl.source.PsiFileImpl.getGreenStub(PsiFileImpl.java:611)
	at org.rust.lang.core.psi.RsFile.hasMacroUseInner(RsFile.kt:261)
	at org.rust.lang.core.resolve2.ModCollector.tryCollectChildModule(ModCollector.kt:219)
	at org.rust.lang.core.resolve2.ModCollector.collectModOrEnumItem(ModCollector.kt:184)
	at org.rust.lang.core.resolve2.ModCollectorBase.collectModOrEnum(ModCollectorBase.kt:172)
	at org.rust.lang.core.resolve2.ModCollectorBase.collectItem(ModCollectorBase.kt:147)
	at org.rust.lang.core.resolve2.ModCollectorBase.collectElementWithoutAttrMacros(ModCollectorBase.kt:102)
	at org.rust.lang.core.resolve2.ModCollectorBase.collectElement(ModCollectorBase.kt:76)
	at org.rust.lang.core.resolve2.ModCollectorBase.collectElements(ModCollectorBase.kt:52)
	at org.rust.lang.core.resolve2.ModCollectorBase.access$collectElements(ModCollectorBase.kt:31)
	at org.rust.lang.core.resolve2.ModCollectorBase$Companion.collectMod(ModCollectorBase.kt:335)
	at org.rust.lang.core.resolve2.ModCollector.collectMod(ModCollector.kt:142)
	at org.rust.lang.core.resolve2.ModCollectorKt.collectExpandedElements(ModCollector.kt:100)
	at org.rust.lang.core.resolve2.DefCollector.recordExpansion(DefCollector.kt:421)
	at org.rust.lang.core.resolve2.DefCollector.expandMacrosInParallel(DefCollector.kt:362)
	at org.rust.lang.core.resolve2.DefCollector.expandMacros(DefCollector.kt:326)
	at org.rust.lang.core.resolve2.DefCollector.collect(DefCollector.kt:85)
	at org.rust.lang.core.resolve2.FacadeBuildDefMapKt.buildDefMap(FacadeBuildDefMap.kt:42)
	at org.rust.lang.core.resolve2.DefMapsBuilder.doBuildDefMap(DefMapsBuilder.kt:121)
	at org.rust.lang.core.resolve2.DefMapsBuilder.access$doBuildDefMap(DefMapsBuilder.kt:19)
	at org.rust.lang.core.resolve2.DefMapsBuilder$buildDefMapAsync$1$1$1.invoke(DefMapsBuilder.kt:101)
	at org.rust.lang.core.resolve2.DefMapsBuilder$buildDefMapAsync$1$1$1.invoke(DefMapsBuilder.kt:100)
	at org.rust.openapiext.UtilsKt$computeInReadActionWithWriteActionPriority$success$1.invoke(utils.kt:377)
	at org.rust.openapiext.UtilsKt$computeInReadActionWithWriteActionPriority$success$1.invoke(utils.kt:376)
	at org.rust.openapiext.UtilsKt.runInReadActionWithWriteActionPriority$lambda$15(utils.kt:372)
	at com.intellij.openapi.application.impl.ApplicationImpl.tryRunReadAction(ApplicationImpl.java:1089)
	at com.intellij.openapi.progress.util.ProgressIndicatorUtils.lambda$runInReadActionWithWriteActionPriority$0(ProgressIndicatorUtils.java:71)
	at com.intellij.openapi.progress.util.ProgressIndicatorUtilService.runActionAndCancelBeforeWrite(ProgressIndicatorUtilService.java:63)
	at com.intellij.openapi.progress.util.ProgressIndicatorUtils.runActionAndCancelBeforeWrite(ProgressIndicatorUtils.java:128)
	at com.intellij.openapi.progress.util.ProgressIndicatorUtils.lambda$runWithWriteActionPriority$1(ProgressIndicatorUtils.java:109)
	at com.intellij.openapi.progress.ProgressManager.lambda$runProcess$1(ProgressManager.java:70)
	at com.intellij.openapi.progress.impl.CoreProgressManager.lambda$runProcess$2(CoreProgressManager.java:186)
	at com.intellij.openapi.progress.impl.CoreProgressManager.lambda$executeProcessUnderProgress$13(CoreProgressManager.java:604)
	at com.intellij.openapi.progress.impl.CoreProgressManager.registerIndicatorAndRun(CoreProgressManager.java:679)
	at com.intellij.openapi.progress.impl.CoreProgressManager.computeUnderProgress(CoreProgressManager.java:635)
	at com.intellij.openapi.progress.impl.CoreProgressManager.executeProcessUnderProgress(CoreProgressManager.java:603)
	at com.intellij.openapi.progress.impl.ProgressManagerImpl.executeProcessUnderProgress(ProgressManagerImpl.java:60)
	at com.intellij.openapi.progress.impl.CoreProgressManager.runProcess(CoreProgressManager.java:173)
	at com.intellij.openapi.progress.ProgressManager.runProcess(ProgressManager.java:70)
	at com.intellij.openapi.progress.util.ProgressIndicatorUtils.runWithWriteActionPriority(ProgressIndicatorUtils.java:106)
	at com.intellij.openapi.progress.util.ProgressIndicatorUtils.runInReadActionWithWriteActionPriority(ProgressIndicatorUtils.java:71)
	at org.rust.openapiext.UtilsKt.runInReadActionWithWriteActionPriority(utils.kt:372)
	at org.rust.openapiext.UtilsKt.computeInReadActionWithWriteActionPriority(utils.kt:376)
	at org.rust.lang.core.resolve2.DefMapsBuilder.buildDefMapAsync$lambda$7(DefMapsBuilder.kt:100)
	at java.base/java.util.concurrent.ForkJoinTask$RunnableExecuteAction.exec(ForkJoinTask.java:1395)
	at java.base/java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:373)
	at java.base/java.util.concurrent.ForkJoinPool$WorkQueue.topLevelExec(ForkJoinPool.java:1182)
	at java.base/java.util.concurrent.ForkJoinPool.scan(ForkJoinPool.java:1655)
	at java.base/java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1622)
	at java.base/java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:165)

```

**Comment by vlad20012** [comment]
The Rust plugin currently does not support the CLion 2023.1 EAP, so it's very not recommended using it there now. We'll introduce 2023.1 EAPs support soon

**Comment by Undin** [comment]
Fixed in #10002. Changes are already available in [nightly](https://plugins.jetbrains.com/plugin/8182-rust/docs/rust-quick-start.html#install-nightly) and will be in the next release.
